src/code/fv-components/bottom-hud.js

```diff
@@ -0,0 +1,20 @@
+import React, {PropTypes} from 'react';
+import TrialCountView from './trial-count';
+
+const BottomHUDView = ({id, className, trial, trialCount}) => {
+
+  return (
```

Comment:
Yes, there will likely be some layout logic, e.g. hiding/showing characters/messages, etc.